customer.go

```diff
@@ -74,6 +75,11 @@ type CustomerRequest struct {
 	PaymentMethodNonce string                    `xml:"payment-method-nonce,omitempty"`
 }
 
+type CustomerUpdateRequest struct {
```

Comment:
What is the difference between `Customer` and `CustomerRequest`?

Reply:
Customer is only used as a return type in requests. The do that in the whole library. Like Subscription/SubscriptionRequest.

I am not sure if I should embed `CustomerRequest` in `CustomerUpdateRequest` as here or make a new type. Feels unnecessary to replicate all the fields though.

Reply:
Understood. It depends how you're going to handle which fields get updated in the `CustomerUpdateRequest` ?